app/views/paye/historicPayAsYouEarn.scala.html

```diff
@@ -136,6 +136,19 @@ <h3 class="heading-large flush break-word-only">@employment.name</h3>
                     <p class="flush--top">
                         <span class="heading-36 faded-text-dark">@withPoundPrefix(MoneyPounds(employment.taxablePayYTD))</span>
                     </p>
+                    @employment.payrollNumber.map{ payrollNumber =>
+                        <p>
+                            @if(employment.isPension) {
+                                <span aria-hidden="true">@Messages("tai.pensionNumber"): @payrollNumber</span>
```

Comment:
This is fine, though you could have a generic function whereby you pass in the message / string

Reply:
done